code-studio/src/js/code-studio.js

```diff
@@ -16,6 +16,8 @@ require('./videos');
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.ColorPicker = require('react-color').default;
```

Comment:
I think I found a way to get this working: https://github.com/code-dot-org/code-dot-org/commit/38e4e5fa9dfbe81ce32df97d74bad8c87ca0b11c.

The problem is that a standard browserify transform does not run on anything in node_modules. For that you need a global transform (-g). It might also be possible to configure browserify-shim to just run globally instead of needing to introduce browserify-global-shim.

I'm not 100% sure I want to introduce radium yet (i.e. this commit may never make it into staging), but it might be worth using this pattern for react-color